src/api/ddoc/views.rst

```diff
@@ -914,3 +917,45 @@ Sending multiple queries to a view
             }
         ]
     }
+
+.. _api/ddoc/view/pagination:
+
+Pagination
+==========
+
+.. versionadded:: 4.0
+
+There is a support for token based pagination. The token based pagination is an
+alternative to previously recomended :ref:`pagination recipe <views/pagination>`
+based on ``offset``, ``limit`` and ``skip``.
+
+The user can request paginated mode by setting ``page_size`` query parameter. When
+pagination is enabled the response would include ``next`` and ``previous`` tokens.
+Which can be used to retrieve next and previous page of the results.
+The maximum possible page size is configured via ``request_limits``
```

Comment:
The above description indicates that "page_size" controls the maximum possible page size, so I gather this means something else. Is this referring to the maximum number of pages?  If so, I suggest rewording to make that more clear:

```suggestion
The maximum number of pages is configured via ``request_limits``
```

If it's something else, can you clarify?